server/routes/support/handlers/offenderDetail.test.ts

```diff
@@ -122,6 +126,17 @@ describe('Route Handlers - Offender detail', () => {
           team: 'The A Team',
           telephone: '078929482994',
         },
+        license: {
+          led: 'Not found',
+          crn: undefined,
```

Comment:
This returning `undefined` is a sign of a problem when the default value that's been set in the `let` is 'Not found'.

Reply:
Done.

Reply:
It still seems to be set to `undefined`. I don't thin CRN is relevant for this section regardless, as it's the Delius identifier rather than a date